### PR TITLE
correct CheckpointContainer to ContainerCheckpoint

### PR DESCRIPTION
**Pull request by pacoxu**
Fixes https://github.com/kubernetes/website/issues/36705

The feature gate name is `ContainerCheckpoint ` according to https://github.com/kubernetes/kubernetes/pull/104907

https://github.com/kubernetes/kubernetes/blob/d569886a234cb65cb1ec777b7ab2e5b9a35d7145/pkg/features/kube_features.go#L189-L194

**BTW**

[reorder feature gates for alphabet order](https://github.com/kubernetes/website/pull/36699/commits/f6cdf2fa48d9c29b186759dbbc21ca8950d8afc2)
The order is wrong for some feature gates and I find a dup and removed it.

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -813,10 +805,16 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 - `ContextualLogging`: When you enable this feature gate, Kubernetes components that support
    contextual logging add extra detail to log output.
-- `ControllerManagerLeaderMigration`: Enables leader migration for
-  `kube-controller-manager` and `cloud-controller-manager`.
+- `ControllerManagerLeaderMigration`: Enables Leader Migration for
+  [kube-controller-manager](/docs/tasks/administer-cluster/controller-manager-leader-migration/#initial-leader-migration-configuration) and
+  [cloud-controller-manager](/docs/tasks/administer-cluster/controller-manager-leader-migration/#deploy-cloud-controller-manager)
+  which allows a cluster operator to live migrate
+  controllers from the kube-controller-manager into an external controller-manager
+  (e.g. the cloud-controller-manager) in an HA cluster without downtime.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:817]: This change really belongs in its own commit. Also, please avoid introducing “e.g.” - we [avoid Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) and their abbreviations.

**Review by sftim — COMMENTED** [review]
Please only change the English localization in this PR. Localization teams will pick up the change to English based on each team's own workflow.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (pacoxu) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: OK.

**Comment by pacoxu** [comment]
> Please only change the English localization in this PR. Localization teams will pick up the change to English based on each team's own workflow.

I reverted Chinese change and order change.
